src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -71,13 +66,12 @@ public Card search(String url) {
       return null;
     }
     String title = getTitle(questionId);
-    String answerBody = getAnswerBody(answerId);
-    if (title == null || answerBody == null) {
+    String description = getAnswerBody(answerId);
+    if (title == null || description == null) {
       return null;
     }
-    // No description or code is allowed for StackOverflow.
-    String description = getDescription(answerBody);
-    String code = getCode(answerBody);
+    // Code is intentionally set to null for better display.
+    String code = null;
     return new Card(title, code, url, description, SOURCE_NAME, ICON_LINK);
```

Comment:
are the other fields (title/description) going to be cleaned here?

Reply:
i think the title is a string that don't need to be cleaned. Also, for description, i cannot think of a certain tag that need to be cleaned at this moment